src/commands/cluster-migrate.json

```
@@ -0,0 +1,55 @@
+{
+    "MIGRATE": {
```

```suggestion
    "MIGRATE-SLOTS": {
```
Based on viktors comment from today that maybe we should specific that it's a slot migration, maybe we should call this migrate-slots? There is a concept of replica migration already.

---

@zuiderkwast @PingXie @enjoy-binbin Any thoughts on making the name slightly more descriptive?

---

My comment? I may have missed it. 😆

When we say migration, we normally mean slot migration. It's the main kind of migration in a cluster. Replica migration is less common (I hope).

I like the current syntax. It's concise and clear enough. SLOTS is not part of the command name but SLOTRANGE is a mandatory option in the syntax.

---

You mentioned something in the weekly meeting about naming ambiguity :) It wasn't specifically about migrations.

---

Oh, I forgot already 😄

---

I am ok with either `MIGRATE` or `MIGRATESLOTS`; I haven't seen a `-` command/sub command so far.

---

Okay so MIGRATESLOTS? I've changed it now